README.md

```diff
@@ -35,5 +35,112 @@ This includes the schema used to set up the info-service.
 ## Docker Compose
 
 This repository contains a [Docker Compose](https://docs.docker.com/compose/)
```

Comment:
Might be worth adding in a point somewhere here if it's the first time the module has been set up, you'll need to pull the submodule otherwise you get:
```
ERROR: .FileNotFoundError: [Errno 2] No such file or directory: '././ci/docker-compose.yml'
```

https://stackoverflow.com/a/1032653